kibbeh/src/modules/search/SearchPage.tsx

```diff
@@ -32,10 +33,15 @@ export const SearchPage: PageComponent<LoungePageProps> = ({}) => {
           <SearchHeader
             onSearchChange={(e) => {
               console.log(e.target.value);
-              conn.query.search(e.target.value).then((r) => setResults(r));
+              setSearchLoading(true);
+              conn.query.search(e.target.value).then((r) => {
+                setResults(r);
+                setSearchLoading(false);
+              });
             }}
             searchPlaceholder="Search"
             onBackClick={() => router.back()}
+            searchLoading={searchLoading}
```

Comment:
use ES6 here

Reply:
Same reason as mentioned above, can only be done on objects.